tensorflow_serving/model_servers/BUILD

```diff
@@ -383,6 +384,7 @@ cc_library(
         "server.cc",
     ],
     hdrs = ["server.h"],
+    textual_hdrs = ["tf_c_api_exported_symbols.lds"],
```

Comment:
This line can be deleted if don't use generating script.

Reply:
deleted